app/models/service.rb

```diff
@@ -5,11 +5,12 @@ class Service < ActiveRecord::Base
   belongs_to :service_upload          # For bulk-uploaded services
   has_many :discontinued_services
 
-  validates_presence_of :recorded_by_educator_id, :student_id, :service_type_id, :recorded_at, :date_started#, :estimated_end_date
+  validates_presence_of :recorded_by_educator_id, :student_id, :service_type_id, :recorded_at, :date_started
+  validate :must_be_discontinued_after_service_start_date
```

Comment:
This validation was originally from the discontinued_service model which I carry over.